x-pack/plugins/observability/public/components/shared/exploratory_view/header/header.test.tsx

```diff
@@ -1,26 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License
- * 2.0; you may not use this file except in compliance with the Elastic License
- * 2.0.
- */
-
-import React from 'react';
-import { render } from '../rtl_helpers';
-import { ExploratoryViewHeader } from './header';
-import * as pluginHook from '../../../../hooks/use_plugin_context';
-
-jest.spyOn(pluginHook, 'usePluginContext').mockReturnValue({
-  appMountParameters: {
-    setHeaderActionMenu: jest.fn(),
-  },
-} as any);
-
-describe('ExploratoryViewHeader', function () {
-  it('should render properly', function () {
-    const { getByText } = render(
-      <ExploratoryViewHeader lensAttributes={{ title: 'Performance distribution' } as any} />
-    );
-    getByText('Refresh');
-  });
```

Comment:
We shouldn't remove the whole test just because it changed the implementation and the existing test didn't apply anymore, we still have to make sure the component is working properly.

Unless there is a good reason to merge it like this, I think we should take the opportunity and improve the testing for the component.

Reply:
Yeah that's fair. The test only checked that the 'Refresh' button was there and since I moved the button I didn't see the point in keeping the test.

Let me think about it